Source/Editor/CustomEditors/Editors/GenericEditor.cs

```diff
@@ -173,16 +173,16 @@ public int CompareTo(object obj)
                             return string.Compare(Display.Group, other.Display.Group, StringComparison.InvariantCulture);
                     }
 
-                    // By declaration order
-                    if (Info.MetadataToken > other.Info.MetadataToken)
-                        return 1;
-                    else if (Info.MetadataToken < other.Info.MetadataToken)
-                        return -1;
-                    else
+                    if(Editor.Instance.Options.Options.General.ScritpMembersOrder == Options.GeneralOptions.MembersOrder.Declaration)
```

Comment:
Typo here, it should probably say `Script`

Reply:
XD fixed